packages/material-ui/src/CssBaseline/CssBaseline.js

```diff
@@ -29,6 +29,12 @@ const useStyles = makeStyles(
           backgroundColor: theme.palette.common.white,
         },
       },
+      '::backdrop': {
```

Comment:
We need to ignore video and dialog. 

```suggestion
      '*:not(dialog):not(video)::backdrop': {
```